Initialise Auth0

Drewbi opened the pull request.

deanvains committed:
Initialising authentication.

Co-authored-by: Drew Alexander <22423862@student.uwa.edu.au>

deanvains committed:
feat(authentication): began working on Authentication through Auth0

Began working on Authentication through Auth0

BREAKING CHANGE: Controllers are currently being changed for authenication

Commit:
Added temp auth

cherbie committed:
feat(auth0 user login): user redirected to login/authenticate themselves using auth0

VueDrawer component gives the user an option to login. The user is redirected to the Auth0 default
login page and authenticated. Successful authentication results in users redirected to home
directory.

Commit:
Merge pull request #43 from codersforcauses/feature/ch-auth-init-ext

feat(auth0 user login): user redirected to login/authenticate themsel…

Commit:
Put things in env

blaiddyd commented on components/other/NavDrawer.vue:
```diff
@@ -36,13 +44,13 @@ export default {
+    }
   }),
   computed: {
     panel: {
```
Is this supposed to be empty?

cherbie committed:
feat(user details): minimal user details displayed in VueDrawer

cherbie committed:
fix(404 error - auth0): callback page for AUTH0 authentication added

Callback page temporarily named login.vue shows status bar for user while authentication is
finalised
